src/helpers/pressable-api-helper.php

```diff
@@ -58,14 +58,18 @@ public static function call_api( string $endpoint, string $method = 'GET', array
 		}
 		if ( 0 !== \strpos( $result['headers']['http_code'], '2' ) ) {
 			$message = '';
-			if ( \property_exists( $result['body'], 'message' ) ) {
-				$message = $result['body']->message;
-			} elseif ( \property_exists( $result['body'], 'error' ) ) {
-				$message = $result['body']->error;
-			}
-
-			if ( \property_exists( $result['body'], 'errors' ) && ! empty( $result['body']->errors ) ) {
-				$message .= ' ' . \implode( ', ', (array) $result['body']->errors );
+			if ( \is_null( $result['body'] ) ) {
+				$message = '{no response body 😭}';
+			} else {
+				if ( \property_exists( $result['body'], 'message' ) ) {
+					$message = $result['body']->message;
+				} elseif ( \property_exists( $result['body'], 'error' ) ) {
```

Comment:
Is there any chance that a response would have both a message and an error, by the api's spec?

Reply:
The API docs only mention the scenario where the error object contain three properties:
* `message`
* `data` (can be null)
* `errors` (can be null)

The scenario where we have `error` instead of `message` is something I found out empirically (can't remember which endpoint(s) though). I am not ruling it out, but according to the specs, we have no reason to believe it can happen.